common/build/eslint-plugin-fluid/src/test/example/no-unchecked-record-access/fileWithOnlyRecordAccess.ts

```diff
@@ -14,14 +14,22 @@ interface NestedIndexProps {
 }
 type StaticType = { a: string; b: string };
 type IndexSignatureType = { [key: string]: string };
+type NullableIndexSignatureType = { [key: string]: string | undefined | null };
+const nullableIndexedRecord: NullableIndexSignatureType = { a: "hello", b: undefined, c: null };
+nullableIndexedRecord.a; // ok: Accessing index property 'a' without requiring a particular result
+nullableIndexedRecord.a.length; // defect: Accessing length of index property 'a', but 'a' might be undefined or null
+nullableIndexedRecord.b?.length; // ok: Using optional chaining to access length safely handles 'undefined'
+nullableIndexedRecord.c?.length; // ok: Using optional chaining to access length safely handles 'null'
 
 const nestedObj: NestedIndexProps = { nested: { a: "hello" } };
 nestedObj.nested.a; // ok: Accessing nested index property 'a' without requiring a particular result
 nestedObj.nested.a.length; // defect: Accessing length of a nested possibly undefined or missing property
 
 const a = "a";
 const b = "b";
-const c = "c";
+/*
+ * The following lint rule should not apply to static types, since they are guaranteed to have the properties they define.
+ */
 const someObjWithStaticType: StaticType = { a: "hello", b: "goodbye" };
 someObjWithStaticType.a; // ok: Accessing string property 'a'
```

Comment:
Comment says "The following lint rule", but what lint rule follows? Perhaps name the lint rule explicitly.